smoke-tests-otel-starter/spring-boot-3.2/src/test/java/io/opentelemetry/spring/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -41,18 +41,19 @@ void restClient() {
                 .body(String.class))
         .isEqualTo("pong");
 
-    if (System.getProperty("org.graalvm.nativeimage.imagecode") != null) {
-      // ignore the trace for creating the db table
-      testing.waitAndAssertTraces(trace -> {}, OtelSpringStarterSmokeTest::assertClient);
-    } else {
-      testing.waitAndAssertTraces(OtelSpringStarterSmokeTest::assertClient);
-    }
-  }
-
-  private static void assertClient(TraceAssert traceAssert) {
-    traceAssert.hasSpansSatisfyingExactly(
-        span -> HttpSpanDataAssert.create(span).assertClientGetRequest("/ping"),
-        span -> span.hasKind(SpanKind.SERVER).hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping"),
-        span -> withSpanAssert(span));
+    // ignore the trace for creating the db table
+    await()
+        .until(
+            () ->
+                testing.getExportedSpans().stream()
+                    .anyMatch(span -> span.getKind() == SpanKind.SERVER));
```

Comment:
I don't get what this is for.

Reply:
It made sense when I added it :upside_down_face: 

I'll remove it and see what happens.